### PR TITLE
feat: add imgix sdk script

### PR DESCRIPTION
**Pull request by frederickfogerty**
This will give consumers of the products integration cartridge access to the js-core library to help them further modify their images.

Example usage in a custom component:
image

with isml as:
```
<img src="${pdict.image_src}" />
```

Results in:
image

### CODE REVIEW

**Comment by frederickfogerty** [comment]
Current dependencies on/for this PR:
* next
  * **PR #278** <a href="https://app.graphite.dev/github/pr/imgix/sf-commerce-cloud/278" target="_blank">Graphite</a>  (this PR)

This comment was auto-generated by [Graphite](https://graphite.dev?utm_source=gh).

File: src/cartridges/int_imgix_products_sfra/cartridge/scripts/imgix/imgix.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+/**
+ * The purpose of this file is to be used by consumers of the imgix integration
+ * to help rendering imgix images.
+ * This file exports url helpers for this purpose.
+ */
+
+import type IImgixClient from "@imgix/js-core";
```
Comment (luqven) [inline src/cartridges/int_imgix_products_sfra/cartridge/scripts/imgix/imgix.ts:7]: This is failing on CI 🧐

**Review by luqven — CHANGES_REQUESTED** [review]
See inline comments

File: src/cartridges/int_imgix_products_sfra/cartridge/scripts/imgix/imgix.ts (hunk 1 above)
Comment (frederickfogerty) [reply src/cartridges/int_imgix_products_sfra/cartridge/scripts/imgix/imgix.ts:7]: Ah, I forgot to check in the dependency change dogscience

**Review by luqven — APPROVED** [review]
Commit lint is being commit lint about that `@` in the commit message 🤦🏼‍♂️. LGTM +1

**Pull request merged**